packages/@webex/internal-plugin-device/src/device.js

```diff
@@ -512,6 +520,7 @@ const Device = WebexPlugin.extend({
           return this.processRegistrationSuccess(response);
         })
         .catch((error) => {
+          console.error(error);
```

Comment:
not sure if this error was intentionally left in?

Reply:
it was not, good catch